pkg/transport/transport.go

```diff
@@ -47,6 +47,13 @@ func NewServer(logger log.Logger, routes []Route, dRoutes []DynamicRoute) *echo.
 	return e
 }
 
+func ReloadRoutes(e *echo.Echo, routes []Route, logger log.Logger) {
+	for _, r := range routes {
+		level.Debug(logger).Log("request_path_added", r.RequestPath)
+		addRoute(e, r.RequestPath, r.Service, logger)
```

Comment:
@amimof 

Thanks for your great work. Overall, it looks good to me.
I have some concerns.

If we use `addRoute` here, does it mean all obsolete routes would remain?

I am not sure is there elegant way to reload the routes at once. At the worst case, we can [list routes](https://echo.labstack.com/guide/routing/#list-routes), and then delete them except `/metrics`, `/debug/pprof/*`, `/config` and `/reload`. It would be great if we can add test a case as well.

Let me know if you have any concern.